client/settings/deposits/index.js

```diff
@@ -0,0 +1,230 @@
+	{ label: __( '24th', 'woocommerce-payments' ), value: 24 },
+	{ label: __( '25th', 'woocommerce-payments' ), value: 25 },
+	{ label: __( '26th', 'woocommerce-payments' ), value: 26 },
+	{ label: __( '27th', 'woocommerce-payments' ), value: 27 },
+	{ label: __( '28th', 'woocommerce-payments' ), value: 28 },
+	{
+		label: __( 'Last day of the month', 'woocommerce-payments' ),
+		value: 31,
+	},
+];
+
+const CustomizeDepositSchedule = () => {
+	const [
+		depositScheduleInterval,
+		setDepositScheduleInterval,
+	] = useDepositScheduleInterval();
+	const [
+		depositScheduleWeeklyAnchor,
+		setDepositScheduleWeeklyAnchor,
+	] = useDepositScheduleWeeklyAnchor();
+	const [
+		depositScheduleMonthlyAnchor,
+		setDepositScheduleMonthlyAnchor,
+	] = useDepositScheduleMonthlyAnchor();
+
+	const handleIntervalChange = ( newInterval ) => {
+		switch ( newInterval ) {
+			case 'weekly':
+				setDepositScheduleWeeklyAnchor(
+					depositScheduleWeeklyAnchor || 'monday'
+				);
+				break;
+
+			case 'monthly':
+				setDepositScheduleMonthlyAnchor(
+					depositScheduleMonthlyAnchor || '1'
+				);
+				break;
+		}
+
+		setDepositScheduleInterval( newInterval );
+	};
+
+	return (
+		<>
+			<div className="schedule-controls">
+				<SelectControl
+					label={ __( 'Frequency', 'woocommerce-payments' ) }
+					value={ depositScheduleInterval }
+					onChange={ handleIntervalChange }
+					options={ [
+						{
+							value: 'daily',
+							label: __( 'Daily', 'woocommerce-payments' ),
+						},
+						{
+							value: 'weekly',
+							label: __( 'Weekly', 'woocommerce-payments' ),
+						},
+						{
+							value: 'monthly',
+							label: __( 'Monthly', 'woocommerce-payments' ),
+						},
+					] }
+				/>
+				{ 'monthly' === depositScheduleInterval && (
+					<SelectControl
+						label={ __( 'Date', 'woocommerce-payments' ) }
+						value={ depositScheduleMonthlyAnchor }
+						onChange={ setDepositScheduleMonthlyAnchor }
+						options={ monthlyAnchors }
+					/>
+				) }
+				{ 'weekly' === depositScheduleInterval && (
+					<SelectControl
+						label={ __( 'Day', 'woocommerce-payments' ) }
+						value={ depositScheduleWeeklyAnchor }
+						onChange={ setDepositScheduleWeeklyAnchor }
+						options={ daysOfWeek }
+					/>
+				) }
+			</div>
+			<p className="help-text">
+				{ 'monthly' === depositScheduleInterval &&
+					__(
+						'Deposits scheduled on a weekend will be sent on the next business day.',
+						'woocommerce-payments'
+					) }
+				{ 'weekly' === depositScheduleInterval &&
+					__(
+						'Deposits that fall on a holiday will initiate on the next business day.',
+						'woocommerce-payments'
+					) }
+				{ 'daily' === depositScheduleInterval &&
+					__(
+						'Deposits will occur every business day.',
+						'woocommerce-payments'
+					) }
+			</p>
+		</>
+	);
+};
+const DepositsSchedule = () => {
+	const depositStatus = useDepositStatus();
+	const completedWaitingPeriod = useCompletedWaitingPeriod();
```

Comment:
See my comment below about `Your first deposit will be held for %s days`.

```suggestion
	const completedWaitingPeriod = useCompletedWaitingPeriod();
	const depositDelayDays = useDepositDelayDays();
```